maintenance_plan_meter_mode/models/maintenance.py

```diff
@@ -14,14 +23,16 @@ class MaintenancePlan(models.Model):
         string='Current value',
         help="This field can be updated manually or "
              "daily adding a value defined.")
+    meter_autoinc_mode = fields.Boolean(
+        string='Manual or External auto increment')
```

Comment:
Since it's not a `selection` field, I think it's better a short description with the `True` (checked) option. In the frontend it's a little bit confusing "Manual _or_ External" for a check, actually is "Self-increased vs Only Manual" (and "Manual" can be through frontend or from and external access e.g. RPC):
```suggestion
    meter_autoinc_mode = fields.Boolean(
        string='Auto incremental mode',
        help='If checked, Current value is periodically incremented according a self-increase value; otherwise only manual changes of Current value are allowed')
```